CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* refactor: remove private function `bigint3_pack` in favor of equivalent `BigInt3::pack86` method [#1329](https://github.com/lambdaclass/cairo-vm/pull/1329)
+
```

Comment:
This is an internal refactor, with no changes to the public api/behaviour so we don't need to include it in the changelog

Reply:
+1

Reply:
Done